pkg/buildinfo/buildinfo.go

```diff
@@ -12,26 +12,25 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package version
+package buildinfo
 
 import "fmt"
 
-// BuildVersion stores the SemVer for the given build
-var BuildVersion string
+// Filled in by build process.
+var (
+	// Version is the version of the code.
+	Version string
 
-// BuildDate stores the date of the build
-var BuildDate string
+	// GitRevision is the commit hash of the code.
+	GitRevision string
```

Comment:
I think the go toolchain embeds this into the binary automatically now?

https://pkg.go.dev/runtime/debug#BuildInfo

Could we grab some of this information from there and remove some of the make machinery?

Reply:
@fasaxc neat! I didn't know about that.

Will need to dig a bit - trivial swap doesn't seem to "just work" - it has empty build information for the `main` module. Potentially something with how we are building.